packages/metrics/src/electron-node/electron-metrics-backend-module.ts

```diff
@@ -0,0 +1,41 @@
+// *****************************************************************************
+// Copyright (C) 2023 STMicroelectronics and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { ContainerModule } from '@theia/core/shared/inversify';
+import { MetricsElectronTokenValidator } from './electron-token-validator';
+import { ElectronTokenValidator } from '@theia/core/lib/electron-node/token/electron-token-validator';
+import { BackendApplicationContribution } from '@theia/core/lib/node';
+import { ExtensionMetricsContribution, MetricsBackendApplicationContribution, MetricsContribution, NodeMetricsContribution } from '../node';
+import { ConnectionHandler, RpcConnectionHandler, bindContributionProvider } from '@theia/core';
+import { measurementNotificationServicePath } from '../common';
+import { MeasurementMetricsBackendContribution } from '../node/measurement-metrics-contribution';
+
+export default new ContainerModule((bind, unbind, isBound, rebind) => {
+    bind(MetricsElectronTokenValidator).toSelf().inSingletonScope();
+    rebind(ElectronTokenValidator).to(MetricsElectronTokenValidator);
```

Comment:
The only difference to the regular backend is the electron token validator rebinding. I would like to suggest moving this into a separate `ContainerModule` and in the `package.json`'s `theiaExtensions` add a new entry only containing the `backendElectron` entry.

This way we could avoid all of the duplicated binding code as the new Container would only contain the electron token validation specifics.

Reply:
Already resolved :+1: